src/pages/config.tsx

```diff
@@ -158,47 +158,59 @@ const ConfigPage: FunctionComponent<ConfigPageProps> = () => {
       {HeaderComponent}
       <section className='e2e-config-page pa4-l bg-snow mw7 center pa4'>
         <h1 className='pa0 f3 ma0 mb4 teal tc'>Configure your IPFS Gateway</h1>
+        <InputSection label='Routing'>
+          <TextInput
+            className='e2e-config-page-input e2e-config-page-input-routers'
+            description='A newline delimited list of delegated IPFS router URLs that expose the /routing/v1 API.'
+            label='Delegated Provider, Peer and IPNS Routing'
+            validationFn={urlValidationFn}
+            onChange={(value) => { setConfig('routers', value) }}
+            value={convertUrlArrayToInput(routers)}
+            preSaveFormat={(value: string) => value.split('\n')}
+            resetKey={resetKey}
+          />
+          <TextInput
+            className='e2e-config-page-input e2e-config-page-input-dnsJsonResolvers'
+            description='A newline delimited list of space delimited key+value pairs for DNS (application/dns-json) resolvers. The key is the domain suffix, and the value is the URL of the DNS resolver. Note: Use only trusted centralized gateways, as they may infer your browsing history.'
```

Comment:
```suggestion
            description='A newline delimited list of space delimited key+value pairs for DNS (application/dns-json) resolvers. The key is the domain suffix, and the value is the URL of the DNS resolver.'
```

Reply:
I think a note here that tracking may be done by the gateways you use is important. Do we want to remove it fully or can we rephrase it?